Error compiling LDC on OSX Mavericks in driver/linker.cpp

```
...
In file included from /Users/dymk/code/ldc/driver/linker.cpp:17:
/Users/dymk/code/ldc/./gen/logger.h:75:16: error: invalid operands to binary expression ('std::ostream' (aka 'basic_ostream<char>') and 'char const[2]')
            OS << Thing;
            ~~ ^  ~~~~~
...
4 warnings and 1 error generated.
make[2]: *** [CMakeFiles/ldc2.dir/driver/linker.cpp.o] Error 1
make[1]: *** [CMakeFiles/ldc2.dir/all] Error 2
make: *** [all] Error 2
```

GCC version:

```
Codeinator:build dymk$ gcc -v
Configured with: --prefix=/Applications/Xcode.app/Contents/Developer/usr --with-gxx-include-dir=/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.9.sdk/usr/include/c++/4.2.1
Apple LLVM version 5.0 (clang-500.2.79) (based on LLVM 3.3svn)
Target: x86_64-apple-darwin13.0.2
Thread model: posix
```

I should mention that I'm using LLVM version 3.2, from the archive linked on the [wiki page on building LDC](http://wiki.dlang.org/Building_LDC_from_source)

I'm not sure if I've configured something incorrectly, or if this is a legit bug in compiling on OSX.

--- a/gen/logger.h
+++ b/gen/logger.h
@@ -16,6 +16,7 @@
 #define LDC_GEN_LOGGER_H
 
 #include <iosfwd>
+#include <iostream>
 
 namespace llvm {
     class Type;

